packages/sdk-cli/src/models/localRelay/index.ts

```diff
@@ -0,0 +1,56 @@
+import { createWriteStream } from 'fs';
+import * as child_process from 'child_process';
+import {
+  relayEntryPointPath,
+  pollRelayInfo,
+  RelayInfo,
+  readRelayInfo,
+  ReadRelayInfoResult,
+} from './relayInfo';
+import { RELAY_LOG_FILE_PATH } from './const';
+
+export { RelayInfo } from './relayInfo';
+
+export async function instance(): Promise<RelayInfo> {
+  // Connect to any existing Relay instance; if doesn't exist, launch a new one.
+  const relayInfo = (await readRelayInfo()) || (await launch());
+
+  if (!relayInfo) {
+    throw new Error("Couldn't obtain Local Relay port and pid from PID file");
+  }
+
+  return relayInfo;
+}
+
+async function launch(): Promise<ReadRelayInfoResult> {
+  const relayJsPath = await relayEntryPointPath();
+
+  // Official docs use fs.openSync, which returns a file descriptor integer.
+  // In our case, using a Writable Stream is more intuitive.
+  const out = createWriteStream(RELAY_LOG_FILE_PATH, { flags: 'a' });
+
+  // Why spawn instead of fork? Fork() doesn't support unref()
+  const relayChildProcess = child_process.spawn('node', [relayJsPath], {
+    detached: true,
+    /**
```

Comment:
Seems like you still have TODOs here to resolve? This comment reads a bit like Node docs, can likely be thinned down a bit once you're done... :)

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/26ee1d9e19334aedfd889137a6271b563b290a65